bootstrap/README.md

```diff
@@ -0,0 +1,11 @@
+# Bootstrap from MLOpsPython repository
+
+To bootstrap from the existing MLOpsPython repository run bootstrap.py script as below
+>python bootstrap.py --d [dirpath] --n [projectname]
+
+This script will download and prepare a directory structure for your project.  
```

Comment:
This should be up top of doc and answer "why do I want to bootstrap?" and what does it do more specifically?  "prepare a directory structure" is not clear.